CHANGELOG.rst

```diff
@@ -11,16 +11,21 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
-1.5.1 - 2020-01-19
-~~~~~~~~~~~~~~~~~~
+[1.6.1] - 2020-01-19
+~~~~~~~~~~~~~~~~~~~~
 
 * Fix the navigation section headers so that they are visible.
 
-1.5.0 - 2020-12-17
-~~~~~~~~~~~~~~~~~~
+[1.6.0] - 2020-12-17
```

Comment:
Last was release was already 1.6.0, but the Changelog used 1.5.0 by mistake, because the real 1.5.0 was never documented.

https://github.com/edx/edx-sphinx-theme/releases